undertow/src/main/java/org/wildfly/extension/undertow/ApplicationSecurityDomainDefinition.java

```diff
@@ -466,6 +467,43 @@ public InputStream getResource(String path) {
             return null;
         }
 
+        private static HttpScope requestScope(HttpServerExchange exchange) {
+            ServletRequestContext servletRequestContext = exchange.getAttachment(ServletRequestContext.ATTACHMENT_KEY);
+
+            if (servletRequestContext != null) {
+                final ServletRequest servletRequest = servletRequestContext.getServletRequest();
+                return new HttpScope() {
+
+                    @Override
+                    public boolean supportsAttachments() {
+                        return true;
+                    }
+
+                    @Override
+                    public void setAttachment(String key, Object value) {
+                        servletRequest.setAttribute(key, value);
+                    }
+
+                    @Override
+                    public Object getAttachment(String key) {
+                        return servletRequest.getAttribute(key);
+                    }
+
+                    @Override
+                    public boolean supportsResources() {
```

Comment:
In this case I think don't add access to resources.  The resources that are being made available here are not specifically in relation to this scope - i.e. they correspond to the whole application and not just the current request.